xilem/src/lib.rs

```diff
@@ -146,7 +146,7 @@ where
         event_loop_runner::run(window_attributes, self.root_widget, self.driver)
     }
 }
-pub trait MasonryView<State, Action = ()>: Send + 'static {
+pub trait MasonryView<State, Action = ()>: Send + Sync + 'static {
```

Comment:
I think requiring `Sync` here makes sense, although we don't require it for anything other than `Arc`s

Reply:
Yeah or views derived from `Arc` as well or anything, that actually is shared between the runtime and the `AppState`, but that'll likely be mostly `Arc`s